src/python_bindings/kmedoids_pywrapper.cpp

```diff
@@ -16,27 +16,36 @@
 #include "kmedoids_pywrapper.hpp"
 
 namespace km {
-PYBIND11_MODULE(banditpam, m) {
-  // Module functions
-  m.doc() = "BanditPAM Python library, implemented in C++";
-  m.attr("__version__") = VERSION_INFO;
-  m.def("get_max_threads",  // TODO(@motiwari): change to get_num_threads
+    PYBIND11_MODULE(banditpam, m
+    ) {
+    // Module functions
+    m.
+
+    doc() = "BanditPAM Python library, implemented in C++";
```

Comment:
??

Reply:
This is due to the `clang-format` and `cpplint` stuff mentioned above. The functionality still remains the same; only the style has changed. This will be fixed in the next release.